composer.json

```diff
@@ -84,11 +84,11 @@
     },
     "scripts": {
         "test": [
-            "vendor/bin/simple-phpunit",
-            "ENCODER=lcobucci vendor/bin/simple-phpunit",
-            "ENCODER=lcobucci ALGORITHM=HS256 vendor/bin/simple-phpunit",
-            "ENCODER=user_id_claim vendor/bin/simple-phpunit",
-            "PROVIDER=lexik_jwt vendor/bin/simple-phpunit"
+            "vendor/bin/simple-phpunit --exclude-group web-token",
+            "ENCODER=lcobucci vendor/bin/simple-phpunit --exclude-group web-token",
+            "ENCODER=lcobucci ALGORITHM=HS256 vendor/bin/simple-phpunit --exclude-group web-token",
+            "ENCODER=user_id_claim vendor/bin/simple-phpunit --exclude-group web-token",
+            "PROVIDER=lexik_jwt vendor/bin/simple-phpunit --exclude-group web-token"
```

Comment:
Ideally this composer script should tests all integrations, but I see that you added a dedicated workflow for this one. 
What is the motivation? If it's only to not require(-dev) the necessary packages then no worry, we can and should require(-dev) them. But maybe I'm missing something else?